docs/transpile/index.mdx

```diff
@@ -0,0 +1,42 @@
+---
+title: Introduction to the transpiler
+description: Introduction to the transpiler
+
+---
+
+
+# Introduction to the transpiler
+
+Transpilation is the process of rewriting a given input circuit to match the topology of a specific quantum device and optimize the circuit instructions for execution on present-day noisy quantum systems.  This documentation covers the tooling and workflows for transpilation which are available to Qiskit users.  If you're using primitives and are only interested in the default transpilation options, look to the section [Configure runtime compilation for Qiskit runtime.](../run/configure-runtime-compilation)
+
+A central component of Qiskit, the transpiler is designed for modularity and extensibility. Its central goal is to be able to easily write new circuit transformations (known as transpiler **passes**), and combine them with other existing passes, allowing you to greatly reduce the depth and complexity of your quantum circuits.  Which passes are chained together and in which order has a major effect on the final outcome. This pipeline is determined by `PassManager` and `StagedPassManager` objects.  The `StagedPassManager` will orchestrate the execution one or more `PassMangers` and determine the order in which they are executed, while the `PassManger` object is merely a collection of one or more passes.  Think of the `StagedPassManager` as the conductor in an orchestra, the `PassManagers` as the different instrument sections, and the `Passes` as the individual musicians.  In this way, you have the power to compose hardware efficient quantum circuits and allow you to execute utility scale work while keeping noise manageable.
+
+
+## Transpiler Stages
+Qiskit utilizes a set of six fundamental stages in the prebuilt transpiler pipeline:
+
+1. `init` - This pass runs any initial passes that are required before we start embedding the circuit to the backend.  This typically involves unrolling custom instructions and converting the circuit to all single- and two-qubit gates. (By default this does nothing)
```

Comment:
```suggestion
1. `init` - This pass runs any initial passes that are required before we start embedding the circuit to the backend.  This typically involves unrolling custom instructions and converting the circuit to all single- and two-qubit gates. (By default this does nothing)
```

Reply:
Need to change the part in parentheses since it's not technically correct.  There's a verification check to see if any control flow operations are valid and unrolls 3-qubit gates to 1 and 2 qubit gates.